vdom/tests/test_core.py

```diff
@@ -27,16 +27,25 @@ def test_to_html_escaping():
     assert div(p("Hello world<script>evil</script>", title='something')).to_html() == '<div><p title="something">Hello world&lt;script&gt;evil&lt;/script&gt;</p></div>'
 
 def test_css():
-    assert div(
-        p('Hello world'),
-        style={
-            'background-color': 'pink',
-            'color': 'white',
-            # Quotes should be entity escaped
-            'font-family': "'something something'"
-        },
-        title='Test'
-    ).to_html() == '<div style="background-color: pink; color: white; font-family: &#x27;something something&#x27;" title="Test"><p>Hello world</p></div>'
+    el = div(
+             p('Hello world'),
+             style={
+                 'backgroundColor': 'pink',
+                 'color': 'white',
+                 # Quotes should be entity escaped
+                 'font-family': "'something something'"
+             },
+             title='Test'
+         )
+    assert el.to_html() == '<div style="background-color: pink; color: white; font-family: &#x27;something something&#x27;" title="Test"><p>Hello world</p></div>'
```

Comment:
I was completely surprised that this escaping actually works! (I tried it in codepen because I was totally skeptical)